### PR TITLE
Make clear when to use trento-premium package

### PR DESCRIPTION
**Pull request by tomschr**
The Trento premium installation package is available for customer that don't have a Kubernetes in place and want to deploy it on K3s on SLES for SAP.

---

Screenshot 2022-03-02 at 13-51-49 Getting started with Trento Premium

---

After the PR is merged, cherry-pick to `maintenance/15-SP3`.

### CODE REVIEW

File: xml/article_sap_trento.xml [hunk 1]
```diff
@@ -293,7 +293,17 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+    <para>
+      The &t.server; is available as package <package>trento-premium</package> in your
+      respective &suse; repository. The package does not require &productname;.
+      This package is available for customers that do not have a K3s; in place and
+      want to deploy the server on K3s for &productname;.
+    </para>
+
+    <para>
+      If you cannot or do not want to install the <package>trento-premium</package>
+      package, install Trento on your &t.server; host as follows: </para>
```
Comment (ajaeger) [inline xml/article_sap_trento.xml:306]: I would say something like: The trento-premium-server-installer simplifies the installation and installs K3s and helm for you, so it replaces the following manual steps.

**Review by ajaeger — CHANGES_REQUESTED** [review]

**Review by abravosuse — COMMENTED** [review]
I don't know how to make references to specific steps or sections.

**Comment by abravosuse** [comment]
@tomschr I've made a suggestion on this PR following your approval request. Do I need to Commit the suggestion? Thx!

**Comment by abravosuse** [comment]
Just following up on this PR @tomschr. Do you need me to commit my suggestion so that you can review it?

**Comment by tomschr** [comment]
@abravosuse Thank you for the suggestions. I've created references to the specific steps in commit ae9821b.

File: xml/article_sap_trento.xml [hunk 2]
```diff
@@ -293,9 +293,34 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+      to deploy the helm chart in your &k8s; cluster.
+      If you don't have a &k8s; cluster or have one but don't want to use it for
+      Trento, you can deploy a small VM (see section 3.1 for minimum requirements
+      and follow steps 1 through 7 to get &t.server; up and running.
+    </para>
+    <para>
+      The &t.server; is available as package <package>trento-premium</package> in your
+      respective &suse; repository. The package does not require &productname;.
+      This package is available for customers that do not have a K3s; in place and
+      want to deploy the server on K3s for &productname;.
```
Comment (ajaeger) [inline xml/article_sap_trento.xml:311]: LInes 308+309 have nothing todo with 310 and 311. Do you want to delete the whole paragraph?

**Review by ajaeger — CHANGES_REQUESTED** [review]

**Pull request merged**